07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,63 @@
+  .nasa-background {
+      background-image: url("../img/nasa-background.jpg");
+      height: 450px;
+      width: 850px;
+      float: top;
+      margin-bottom: 20px;
+  }
+  h1 {
+      color: white;
+      text-align: center;
+      padding-top: 150px;
+      font-family: sans-serif;
+      font-weight: bold;
+      font-size: 75px;
+  }
+  .clearfix:after {
+      visibility: hidden;
+      display: block;
+      font-size: 0;
+      content: " ";
+      clear: both;
+      height: 0;
+  }
+  main {
```

Comment:
consider adding a padding to the container of your floated items. in this way, you only have to add space to the left and right of the middle block.